github.go

```diff
@@ -84,7 +84,13 @@ func NewGitHubClient(source Source) (*GitHubClient, error) {
 			return nil, err
 		}
 
-		clientV4 = githubv4.NewEnterpriseClient(source.GitHubAPIURL+"graphql", httpClient)
+		var v4URL string
+		if strings.Contains(source.GitHubAPIURL, "v3") {
```

Comment:
Should't this be

```
if strings.HasSuffix(source.GitHubAPIURL, "/v3/") {
  v4URL = source.GitHubAPIURL[:len(input)-4] + "/graphql"
} else {
  return nil, fmt.Errorf(...)
}
```

Reply:
Changed to `HasSuffix()` 👍 

Not sure if it should return an error for the `else` case. Can you explain why? I don't think it should return an error based on how the public github api is structured. Their v3 api endpoint is simply `api.github.com` with no `/v3/` appended. Not sure if enterprise github api endpoints can/do follow the same structure.